### PR TITLE
WIP Inspector config tests

### PR DESCRIPTION
**Pull request by makyo**

### CODE REVIEW

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3139/
Test FAILed.

File: jujugui/static/gui/src/app/components/boolean-config/test-boolean-config.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    YUI().use('boolean-config', () => { done(); });
+  });
+
+  it('renders checked boolean options', () => {
+    var option = {
+      key: 'foo',
+      type: 'boolean',
+      value: true
+    }
+    var shallowRenderer = testUtils.createRenderer();
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/boolean-config/test-boolean-config.js]: This is a little verbose so I created a utils wrapper in jsTestUtils which saves a step.

File: jujugui/static/gui/src/app/components/boolean-config/test-boolean-config.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        config={false} />);
+    var output = shallowRenderer.getRenderOutput();
+    assert.equal(output.props.className, 'boolean-config');
+    assert.equal(
+      output.props.children[1].props.children[0].props.checked,
+      false);
+  });
+
+  it('handles change events', () => {
+    // TODO
```
Comment (makyo) [inline jujugui/static/gui/src/app/components/boolean-config/test-boolean-config.js]: Note to self

to do the onChange I'd grab the actual onChange method and call it
hatch
so for example
hatch
output.props.children[0].props.onChange()

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3157/
Test FAILed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3188/
Test FAILed.

**Pull request closed without merging**